generators/react/templates/src/main/webapp/app/modules/account/password/password.tsx.ejs

```diff
@@ -72,48 +73,56 @@ export const PasswordPage = () => {
               Password for {account.login}
             </Translate>
           </h2>
-          <ValidatedForm id="password-form" onSubmit={handleValidSubmit}>
-            <ValidatedField
-              name="currentPassword"
-              label={translate('global.form.currentpassword.label')}
-              placeholder={translate('global.form.currentpassword.placeholder')}
+          {/* eslint-disable-next-line @typescript-eslint/no-misused-promises */}
+          <Form id="password-form" onSubmit={ handleSubmit(handleValidSubmit) }>
+            <ValidatedTextInput
+              register={ register }
+              touchedFields={ touchedFields }
+              errors={ errors }
+              setValue={ setValue }
+              nameIdCy="currentPassword"
+              validate={{ required: translate('global.messages.validate.newpassword.required') }}
+              labelPlaceholderKey="global.form.currentpassword.label"
+              inputPlaceholderKey="global.form.currentpassword.placeholder"
```

Comment:
I believe the original code `label={translate('global.form.newpassword.label')}` still works well. The code volume remains roughly the same, and it allows for direct text without using i18n for the label.